source/includes/usage-examples/sample-app.py

```diff
@@ -1,9 +1,10 @@
+import pymongo
```

Comment:
Q: is this line needed? I have an app that works with just `from pymongo import MongoClient`

Reply:
Yeah it's needed for the classes in the `bulk_write()` method. It'll also be necessary for indexes which uses the sample sample app